pkg/agents/team4/trainingAgent/getLifeExpectancy.py

```diff
@@ -17,58 +17,62 @@
 for i in range(len(lines)):
     logs.append(json.loads(lines[i]))
 
```

Comment:
MINOR: `logs = [ json.loads(lines[i]) for i in range(len(lines)) ]` - just a suggestion non-necessary.

Reply:
the training agent was just for us and won't be used in the final simulation :)